code/game/objects/structures/dresser.dm

```diff
@@ -5,6 +5,8 @@
 	icon_state = "dresser"
 	density = 1
 	anchored = 1
+	burn_state = FLAMMABLE
+	burntime = 25
```

Comment:
One almost has to wonder if you even want these buildable. It would be a nice QoL being able to change your undies whenever and wherever you want (heh) but I could think of a few places where one of these would be better used. And then you have to wonder what bluespace put all the other undies in there once one is built